app/(landing)/components/CTAbtn.tsx

```diff
@@ -66,8 +65,8 @@ const CTAbtn = ({
                     </Button>
                 ) : (
                     <Form {...form}>
-                        <form onSubmit={form.handleSubmit(onSubmit)} className='flex gap-2 mt-[68px] justify-center items-center'>
-                            <div className="flex rounded-xl">
+                        <form onSubmit={form.handleSubmit(onSubmit)} className='flex flex-col gap-0 md:gap-4 mt-4 md:mt-12 md:flex-row md:items-center '>
```

Comment:
I guess by default the gap is 0, so we don't need to add a class explicitly, right?

```suggestion
                        <form onSubmit={form.handleSubmit(onSubmit)} className='flex flex-col md:gap-4 mt-4 md:mt-12 md:flex-row md:items-center'>
```